src/lib/app-routes/common/auth/keys.js

```diff
@@ -44,9 +44,9 @@ exports.register = function(server, options, next) {
 
     const LoginRecord = server.plugins.bookshelf.model('LoginRecord');
 
-    server.state(config.get('casCookieName'), {
+    server.state(casCookieUtils.cookieName, {
         path: '/',
-        domain: config.get('cookieDomain')
+        domain: config.get('cookieDomain'),
```

Comment:
@dylancwood, did you determine that trailing commas were desired/permissible/undesired at the latest airbnb spec?  i vaguely call that we permit them now, as of a couple short months ago?